ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.14-ec.yaml

```diff
@@ -54,7 +54,7 @@ tests:
     - chain: openshift-qe-data-path-tests
     workflow: rosa-aws-sts
   timeout: 5h0m0s
-- as: perfscale-rosa-multiaz-ec-ci-tests-2nodes-periodic
+- as: perfscale-rosa-multiaz-ec-ci-tests-24nodes-periodic
```

Comment:
Can the test name here include `data path` or `control plane` or something of that sort for more clarity?

Reply:
@svetsa-rh Ack, once this PR gets merged - will change the PR details as well:
https://github.com/openshift/release/pull/42788